sources/packages/backend/apps/api/src/route-controllers/student-loan-balance/student-loan-balance.aest.controller.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  ParseIntPipe,
+} from "@nestjs/common";
+import { ApiNotFoundResponse, ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
```

Comment:
Pretty minor and not a blocker, and maybe not part of the PR, we introduced an index under auth and we never actually started to use it.
All auth-related imports can be under one single index.

Reply:
Sure I will do this change in the upcoming PR.